.goreleaser.yml

```diff
@@ -0,0 +1,23 @@
+builds:
+  - main: main.go
+    binary: kube-bench
+    goos:
+      - darwin
+      - linux
+    goarch:
+      - amd64
+# Archive customization
+archive:
+  format: tar.gz
+dockers:
+  -
+    binary: kube-bench
+    image: aquasecurity/kube-bench
```

Comment:
This is referring to our image on Docker Hub, right? The repo name is aquasec/kube-bench (aquasec not aquasecurity)

Reply:
Fixed!